exir/passes/replace_custom_ops_with_aten_ops_pass.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# pyre-unsafe
+
+from typing import Dict
+
+import torch
+from executorch.exir.pass_base import ExportPass
+from executorch.extension.llm.custom_ops import preprocess_custom_ops  # noqa
+from torch._ops import OpOverload
+
+_REPLACE_CUSTOM_OP_WITH_ATEN_OP: Dict[OpOverload, OpOverload] = {
+    torch.ops.preprocess.pad.default: torch.ops.aten.constant_pad_nd.default,
```

Comment:
I don't think the whole file should live inside `exir/passes`. How about putting it under `examples/models/flamingo/passes`?

Reply:
yep - updated